doc/portfile.7

```diff
@@ -655,26 +711,33 @@ Port dependencies should refer to other MacPorts ports
 whenever possible, therefore each dependency should be
 expressed in the format:
 .Bl -tag -width ls
-.It port: Ns Ao port Ac
+.It Li port: Ns Aq port
```

Comment:
Does it make sense to add some indentation here?

Reply:
Done.